.travis.yml

```diff
@@ -24,9 +33,9 @@ before_script:
 
   # Install composer dependencies
   - composer validate
-  - composer require --no-update silverstripe/installer 4.0.x-dev
-  - if [[ $DB == PGSQL ]]; then composer require --no-update silverstripe/postgresql 2.0.x-dev; fi
-  - composer install --prefer-dist --no-interaction --no-progress --no-suggest --optimize-autoloader --verbose --profile
+  - composer require --no-update silverstripe/installer:^4
+  - if [[ $DB == PGSQL ]]; then composer require --no-update silverstripe/postgresql:^2; fi
+  - composer install --prefer-source --no-interaction --no-progress --no-suggest --optimize-autoloader --verbose --profile
```

Comment:
Why are you changing this? It doubles the build time in this module (comparing the previous master build against this PR's build). In the past we've invested time in optimising build times, and this feels like a definite step backwards.

Reply:
yeah, that's a leftover from the first version where we've been using framework configs for phpcs